js/src/views/ParityBar/parityBar.js

```diff
@@ -45,12 +46,31 @@ class ParityBar extends Component {
     }
 
     if (count < newCount) {
-      this.setState({ opened: true });
+      this.setOpened(true);
     } else if (newCount === 0 && count === 1) {
-      this.setState({ opened: false });
+      this.setOpened(false);
     }
   }
 
+  setOpened (opened) {
+    this.setState({ opened });
+
+    if (!this.bar) {
+      return;
+    }
+
+    // Fire up custom even to support having parity bar iframed.
+    const event = new CustomEvent('parity.bar.visibility', {
+      bubbles: true,
+      detail: { opened }
+    });
+    this.bar.dispatchEvent(event);
```

Comment:
newline before. (As per slightly stricter rules - things are growing, just trying to keep consistency going)